dcpy/test/connectors/edm/recipes/test_recipes.py

```diff
@@ -118,10 +118,41 @@ def test_import_dataset(self):
         )
         recipes.import_dataset(ds, pg_mock)
 
+        assert (
+            pg_mock.insert_dataframe.called
+        ), "PostgresClient.insert_dataframe should be called"
+
         # Verify the values that PostgresClient.insert_dataframe was called with
         (
             df_inserted_actual,
             table_insert_name_actual,
         ) = pg_mock.insert_dataframe.call_args_list[0][0]
         assert df_inserted_actual.equals(_mock_preprocessor(ds.name, _test_df))
         assert table_insert_name_actual == ds.import_as
+
+    def test_import_parquet(self):
+        self.set_mocks()
+        pg_mock = MagicMock()
+        ds = recipes.Dataset(
+            name="test",
+            version="1",
+            import_as="new_table_name",
+            file_type=recipes.DatasetType.parquet,
+            preprocessor=recipes.DataPreprocessor(module="dcpy", function="preproc"),
+        )
+
+        def _mock_fetch_parquet(ds, _):
+            out_path = TEMP_DATA_PATH / f"{ds.name}.parquet"
+            _test_df.to_parquet(out_path, index=False)
+            return out_path
```

Comment:
maybe worth moving this to line 98ish to group it with `_mock_fetch_dataset`?

Reply:
yep, good call. Fixed in latest commit